x-pack/plugins/alerting/server/application/maintenance_window/methods/update/update_maintenance_window.ts

```diff
@@ -62,7 +64,8 @@ async function updateWithOCC(
         buildEsQuery(
           undefined,
           [{ query: scopedQuery.kql, language: 'kuery' }],
-          scopedQuery.filters as Filter[]
+          scopedQuery.filters as Filter[],
+          esQueryConfig
```

Comment:
Same comment as for `create_maintenance_window.test.ts` I took a quick look at the tests but couldn't make much sense of it 😅